pkg/odo/cli/delete/component.go

```diff
@@ -42,6 +63,28 @@ func (o *ComponentOptions) Run() error {
 
 // deleteNamedComponent deletes a component given its name
 func (o *ComponentOptions) deleteNamedComponent() error {
+	log.Info("Searching resources to delete, please wait...")
+	list, err := o.clientset.DeleteClient.ListResourcesToDelete(o.name, o.clientset.KubernetesClient.GetCurrentNamespace())
+	if err != nil {
+		return err
+	}
+	if len(list) == 0 {
+		log.Infof("No resource found for component %s\n", o.name)
```

Comment:
This could be a warning instead. Also adding a success message saying "Deletion completed" or similar would help.